core/src/main/java/org/kiwix/kiwixmobile/core/search/SearchScreen.kt

```diff
@@ -314,27 +318,34 @@ private fun SearchListItem(
 ) {
   Row(
     modifier = Modifier
-      .fillMaxSize()
+      .fillMaxWidth()
+      .height(IntrinsicSize.Min)
       .padding(horizontal = EIGHT_DP)
       .padding(top = SEVEN_DP)
       .background(
         shape = RoundedCornerShape(EIGHT_DP),
         color = MaterialTheme.colorScheme.onBackground.copy(alpha = 0.05f)
-      ),
+      )
+      .clip(RoundedCornerShape(EIGHT_DP)),
```

Comment:
@atharvyadav22 Why are we again adding this? There is already a `shape` added in the `background`.